test/jsonapi/plugs/query_parser_test.exs

```diff
@@ -67,12 +67,20 @@ defmodule JSONAPI.QueryParserTest do
     assert filter[:name] == "jason"
   end
 
-  test "parse_filter/2 handles multiple comma-separated filter values" do
+  test "parse_filter/2 handles multiple comma-separated filter values if configured" do
+    Application.put_env(:jsonapi, :filter_values_separator, ";")
```

Comment:
Should this be cleaned up in an [`on_exit`](https://hexdocs.pm/ex_unit/master/ExUnit.Callbacks.html#on_exit/2) hook?

Reply:
Absolutely! I've restored a previous commit where the whole env is backed up before tests and restored on exit.
This way we are sure we won't forget to clean up other variables.